english/dragons/fire-everywhere.md

```diff
@@ -42,7 +42,11 @@ Drogon to fly...
 You do not much energy left in you...You are extremely tired...You don't even realize when you 
 fell asleep...
 
-When you wake up its noon and you come to know that it was all a dream !
+When you wake up its noon and you and find yourself lying face to face with [tyrion lanister](http://gameofthrones.wikia.com/wiki/Tyrion_Lannister)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.